src/utils/switch-site.ts

```diff
@@ -28,8 +28,13 @@ export async function loginToSite(_site: SiteKey | null) {
     password: "password",
   })
 
+  // Wait for the session to be updated.
+  await delay(100)
```

Comment:
I was getting all sorts of errors without this delay, even though the `await login(...)` above has been awaited.